api-module-library/microsoft-teams/manager.js

```diff
@@ -32,7 +32,7 @@ class Manager extends ModuleManager {
             client_secret: process.env.TEAMS_CLIENT_SECRET,
             redirect_uri: `${process.env.REDIRECT_URI}/microsoft-teams`,
             tenant_id: process.env.TENANT_ID,
```

Comment:
Eventually, this would need to be retrieved via an API request/token generation. Once generated, we get the tenant ID, which then gets stored with the Credential/Entity. ("Eventually" means once the app is "Public" and available to be installed on multiple teams)